### PR TITLE
updated README with more detailed setup instructions

### PR DESCRIPTION
**Pull request by schwarrrtz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,24 @@
...
+1. in `/etc/postgresql/<VERSION>/main/pg_hba.conf` change 'local ... all ... all ... peer' to 'local ... all ... all ... md5'
+
+1. copy .env.example to .env and fill out variables appropriately for your local environment
+
+1. run `source .env`
+
+1. run `sudo ./setup_postgresql.sh`
+
+1. to check that everything is working, run `npm test` and check that the tests pass.
+
```
Comment (trisapeace) [inline README.md:21]: Can we include a section about how to run the dashboard? It was something with `server.js`, right?

File: README.md [hunk 2]
```diff
@@ -8,13 +27,29 @@
...
+1. plug the RPi into a router that you have admin access to and determine its IP address
+
+1. ssh into the raspberry pi and clone this repository.
+
+# How to set up the pairing tool
+
+1. follow the generic RPi setup instructions above
+
+1. cd into the `pairing-tool` directory
```
Comment (trisapeace) [inline README.md:46]: Can we make this `BraveButtons/pairing-tool`?

Comment (trisapeace) [reply README.md]: I did this.

File: README.md (hunk 1 above)
Comment (schwarrrtz) [reply README.md:21]: done.

**Comment by schwarrrtz** [comment]
LGTM!

**Comment by trisapeace** [comment]
Task linked: [CU-aekyav pi setup script should install pip3](https://app.clickup.com/t/aekyav)

**Pull request merged**